scss/_patterns_notifications.scss

```diff
@@ -205,43 +219,73 @@ $notification-text-margin-bottom: $spv--large - $spv-nudge;
 @mixin vf-notifications-default {
   .p-notification {
     @extend %vf-notification;
-    @include vf-highlight-bar($color-information, left, true);
-    @include vf-icon-info($color-mid-dark);
+    @include vf-highlight-bar($colors--theme--border-information, left, true);
+    @include vf-icon-info(#24598f);
```

Comment:
What is this colour? Is it from any variable? if so, please refer the variable and not hardcode the color value.